features/authentication/use-logout.ts

```
@@ -27,21 +30,39 @@ export const useLogout = () => {
+                userId: currentUser.id,
+              })
+              await Promise.all(
+                currentUserIdentities.map((identity) =>
+                  unlinkIdentityFromDeviceMutation({
+                    identityId: identity.id,
+                    deviceId: currentDevice.id,
+                  }),
+                ),
+              )
+              resolve()
+            }),
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)